{"name": "test_fails", "status": "failed", "statusDetails": {"message": "AssertionError: assert 2 == 1", "trace": "def test_fails():\n>       assert 2 == 1  # keep one failing to see FAIL in results\n        ^^^^^^^^^^^^^\nE       assert 2 == 1\n\ntests\\test_smoke.py:5: AssertionError"}, "start": 1759507328872, "stop": 1759507328872, "uuid": "281410b4-a48f-46ef-87a6-c8bd3fa194c0", "historyId": "654d579774cbafe816589902b7739b1b", "testCaseId": "654d579774cbafe816589902b7739b1b", "fullName": "tests.test_smoke#test_fails", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_smoke"}, {"name": "host", "value": "סלווא_נעום"}, {"name": "thread", "value": "6012-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_smoke"}], "titlePath": ["tests", "test_smoke.py"]}